src/domdiv/draw.py

```diff
@@ -1486,7 +1486,7 @@ def drawTab(self, item, panel=None, backside=False):
             # adjust dimensions based on the application image metrics
             bannerHeight += cardType.getTabTextHeightOffset() * tabScale
             # adjust for space around banners and scalloped edges
-            margin = tabWidth / 18 if card.isExpansion() else tabWidth / 48
+            margin = tabWidth / 18
```

Comment:
I don't think this change works as intended. The margin is the distance from the edge of the tab to the artwork. The `tabWidth / 18` setting pushes the artwork inward so that it doesn't clash with the scalloped edges on the expansion dividers. The normal banner tabs have more room, so the `tabWidth / 48` pushes the artwork closer to the edge. Changing it to `tabWidth / 18` actually leaves _less_ room for text.

If you want to leave more room for text, the place to change that is the `padding` variable, which is initialized a few lines above:
```
# whitespace
margin = padding = 2
```

Reply:
I'm curious which case prompted this change. The minimum 2mm padding _should_ prevent the text from ever running into the artwork. The only exceptions that I know are about are cases where the text is too wide to fit, even after breaking it into two lines and shrinking to the minimum allowed font size. In those cases, this change will make things worse because the margin goes outside the artwork, and `tabWidth / 18` is wider than `tabWidth / 48`.

Reply:
Maybe I misinterpreted, but before this change the text was running into the left side of the artwork on the banners if I turned off the cost icon, and always was for dividers that don't have a cost (Traits, for example). It was a quick fix with me trying to interpret old and changed code seeing how `margin` was used, so feel free to suggest a better one. Looked ok to me after the fix for the settings combinations I tried.

Reply:
Oh! Your comment here inspired me to look at all of the graphics with & without the cost icons, and I actually like your margins better for everything _except_ the cost icons. That's good because it means that it should be easy to adjust without any further type attributes. Instead, the cost icons can outdent a bit when they're combined with artwork. I'll fix that in my next PR. Thanks!

Reply:
On further investigation, the 1/18 margin is quite close to the card art! If anything, the card margins are a bit wider on the coin side. The banners are 52.5mm wide, and domdiv adds a 1mm cushion around that for a total of 54.5mm. If we made a tab the same width as the cards, 1/18 = ca. 3mm from the edge of the tab, or 2mm from the edge of the banner.

The coins on the cards are set 2.5mm from the left end of the banner, and the set icons are set 1.5mm from the right end, and honestly the set icons are a bit cramped on the cards. Therefore it actually makes sense to use a margin 1/18 of the tab width! I don't know where I got the 1/48 value from earlier. I wonder if I was looking at an outlier, or perhaps the graphics from one of the digital adaptations?

There's one small flaw with the `tabWidth / 18` calculation. It scales the margin with the entire `tabWidth` even though there's always a fixed 1mm cushion around the artwork. It should be something like this instead:
```
    margin = 1 + (tabWidth - 2) * 2.5 / 52.5
```
I'll fix that in my followup PR.